src/runtime/storage/store-interface.ts

```diff
@@ -200,6 +194,17 @@ export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCo
     }
   }
 
+  async idle(): Promise<void> {
+    return Promise.resolve();
+  }
+
+  async serializeContents(): Promise<T['data']> {
+    throw new Error('Method not implemented.');
```

Comment:
maybe something more descriptive such as "ActiveMuxer contents cannot be serialized"

Reply:
Done.